README.md

```diff
@@ -0,0 +1,7 @@
+# Examples of IDPartner usage without the node-oidc-client library
+
+This repository contains 2 examples:
+1. client-secret-basic-example: A node project which obtains a token using the client_secret_basic authentication method
+   Please check [here](./client-secret-basic-example/README.md) for further details.
+1. tls-client-auth-example: A node project which obtains a token using mTLS.
```

Comment:
should we call this `Mutual TLS`?  not sure devs will understand tls-client-auth